### PR TITLE
updating some links to the product site

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: N/A

Description of Changes:
* Fixing some links to the product site

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: enterprise/server/dockerservices.md [hunk 1]
```diff
@@ -87,9 +87,9 @@ Proceed to setting up and deploying the containerized services:
 
 The following sections assist with containerizing the {{ site.productname }} services for users with licenses to self-hosted versions of:
 
-* [_Tiny Enhanced Media Embed_](https://apps.tiny.cloud/products/enhanced-media-embed/)
-* [_Tiny Link Checker_](https://apps.tiny.cloud/products/link-checker/)
-* [_Tiny Spell Checker Pro_](https://apps.tiny.cloud/products/spell-checker-pro/)
+* [_Tiny Enhanced Media Embed_]({{site.plugindirectory}}enhanced-media-embed/)
+* [_Tiny Link Checker_]({{site.plugindirectory}}link-checker/)
+* [_Tiny Spell Checker Pro_]({{site.plugindirectory}}spell-checker-pro/)
```
Comment (lnewson) [inline enterprise/server/dockerservices.md:92]:
```suggestion
* [_Tiny Enhanced Media Embed_]({{site.plugindirectory}}enhanced-media-embed)
* [_Tiny Link Checker_]({{site.plugindirectory}}link-checker)
* [_Tiny Spell Checker Pro_]({{site.plugindirectory}}spell-checker-pro)
```
The site doesn't have a trailing `/` when testing this.

File: integrations/wordpress.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ To use {{site.productname}} in WordPress, TinyMCE Advanced is a great option.
 
 ## PowerPaste for WordPress
 
-[{{site.productname}} PowerPaste for WordPress]({{site.productpages}}/powerpaste/) is a premium plugin from the makers of {{site.productname}} that brings the PowerPaste features to WordPress. It automatically fixes Word formatting, creates tables from Excel, and links & embed images with content pasted from the web.
+[{{site.productname}} PowerPaste for WordPress]({{site.plugindirectory}}powerpaste/) is a premium plugin from the makers of {{site.productname}} that brings the PowerPaste features to WordPress. It automatically fixes Word formatting, creates tables from Excel, and links & embed images with content pasted from the web.
```
Comment (lnewson) [inline integrations/wordpress.md:16]:
```suggestion
[{{site.productname}} PowerPaste for WordPress]({{site.plugindirectory}}powerpaste) is a premium plugin from the makers of {{site.productname}} that brings the PowerPaste features to WordPress. It automatically fixes Word formatting, creates tables from Excel, and links & embed images with content pasted from the web.
```

File: release-notes/release-notes503.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ Currently, there are three types of formats available:
 
 * **Title Case** - The **Title Case** option changes anything other than articles, coordinating conjunctions, or short prepositions to capital letters. The default rule set for **Title Case** is based on [Wikipedia Title Case](https://titlecaseconverter.com/rules/#WP).
 
-To try out **Case Change** start with this [dedicated product page]({{site.productpages}}/case-change/) to see all the flexible pricing options.
+To try out **Case Change** start with this [dedicated product page]({{site.plugindirectory}}case-change/) to see all the flexible pricing options.
```
Comment (lnewson) [inline release-notes/release-notes503.md:30]:
```suggestion
To try out **Case Change** start with this [dedicated product page]({{site.plugindirectory}}case-change) to see all the flexible pricing options.
```

File: release-notes/release-notes507.md [hunk 4]
```diff
@@ -37,7 +37,7 @@ For more information on new toolbar button and menu item refer to the [documenta
 
 The new **Premium Skins and Icon Packs** adds an easy way to quickly give TinyMCE a new look, or to make it easier to integrate with popular CSS frameworks, such as Bootstrap, Material Design from Google, and Fabric, which is the design system for Microsoft Office.
 
-To try out the **Premium Skins and Icon Packs** start with this [dedicated product page]({{site.productpages}}/skins-and-icon-packs/) or check out the [pricing page](https://www.tiny.cloud/pricing) to see all the flexible pricing options.
+To try out the **Premium Skins and Icon Packs** start with this [dedicated product page]({{site.plugindirectory}}skins-and-icon-packs/) or check out the [pricing page](https://www.tiny.cloud/pricing) to see all the flexible pricing options.
```
Comment (lnewson) [inline release-notes/release-notes507.md:40]:
```suggestion
To try out the **Premium Skins and Icon Packs** start with this [dedicated product page]({{site.plugindirectory}}skins-and-icon-packs) or check out the [pricing page](https://www.tiny.cloud/pricing) to see all the flexible pricing options.
```

**Review by lnewson — APPROVED** [review]

File: _includes/release-notes/51premium-plugins.md [hunk 5]
```diff
@@ -16,7 +16,7 @@ The plugin is capable of sorting:
 
 For information on this plugin, see:
 
-* [The Advanced Tables product page](https://apps.tiny.cloud/products/advanced-tables/).
+* [The Advanced Tables product page]({{site.plugindirectory}}advanced-tables/).
```
Comment (lnewson) [inline _includes/release-notes/51premium-plugins.md:19]:
```suggestion
* [The Advanced Tables product page]({{site.plugindirectory}}advanced-tables).
```

File: advanced/creating-an-icon-pack.md [hunk 6]
```diff
@@ -23,7 +23,7 @@ A {{site.productname}} icon pack is a `.js` file containing strings of [SVG's](h
 
 An icon pack only requires the custom icons to be included; the default {{site.productname}} icons are used as a fallback for icons missing from the custom icon pack.
 
-> Don't forget to explore our ready-to-use Premium Icon Packs such as 'Material' icons, and a smaller version of our default icons at [{{site.prem_skins_icons}}]({{site.productpages}}/skins-and-icon-packs/).
+> Don't forget to explore our ready-to-use Premium Icon Packs such as 'Material' icons, and a smaller version of our default icons at [{{site.prem_skins_icons}}]({{site.plugindirectory}}skins-and-icon-packs/).
```
Comment (lnewson) [inline advanced/creating-an-icon-pack.md:26]:
```suggestion
> Don't forget to explore our ready-to-use Premium Icon Packs such as 'Material' icons, and a smaller version of our default icons at [{{site.prem_skins_icons}}]({{site.plugindirectory}}skins-and-icon-packs).
```

File: enterprise/paste-from-word.md [hunk 7]
```diff
@@ -7,7 +7,7 @@ keywords: enterprise powerpaste power paste powerpaste_word_import powerpaste_ht
 
 ## Improve your productivity & writing experience
 
-[{{site.productname}} PowerPaste]({{site.productpages}}/powerpaste/) automatically cleans up content from Microsoft Word and other HTML sources to ensure clean, compliant content that matches the look and feel of your site or app.
+[{{site.productname}} PowerPaste]({{site.plugindirectory}}powerpaste/) automatically cleans up content from Microsoft Word and other HTML sources to ensure clean, compliant content that matches the look and feel of your site or app.
```
Comment (lnewson) [inline enterprise/paste-from-word.md:10]:
```suggestion
[{{site.productname}} PowerPaste]({{site.plugindirectory}}powerpaste) automatically cleans up content from Microsoft Word and other HTML sources to ensure clean, compliant content that matches the look and feel of your site or app.
```

**Review by EkimChau — APPROVED** [review]

**Pull request merged**